svgo.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+for i in $(git diff  --cached --diff-filter=A --name-only|grep .svg)
```

Comment:
Kjører dette scriptet for alle OS, og trenger en installere noe pakke lokalt på egen maskin. Ser de andre pre-commit hooks Husky og pretty-quick kommer fra package.json, men ikke svgo.